NitroxServer/ConsoleCommands/LoginCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Linq;
+using NitroxModel.Logger;
+using NitroxServer.Communication;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+
+namespace NitroxServer.ConsoleCommands
+{
+    internal class LoginCommand : Command
+    {
+        private readonly PlayerManager playerManager;
+
+        public LoginCommand(PlayerManager playerManager) : base("login")
```

Comment:
You can add a `ServerConfig` parameter here like the `PlayerManager` and store it in a private readonly field. Make sure ServerConfig has a property to get the password. Then use it in `RunCommand` to do the check.